vagrant/provisioning/arkcase-app-upgrade.yml

```diff
@@ -13,5 +13,7 @@
       tags: [foia]
     - role: arkcase-post-upgrade
       tags: [core]
+    - role: start-arkcase
```

Comment:
I see we are mixing up changes from another ticket in this pull request.  Also, 'start-arkcase' should always be last.

Reply:
you should remove these commits or revert these changes from this pull request, since you already applied these changes in the other pull request.

Reply:
@kikomanaskov333 it seems like you still have to fix or remove these `start-arkcase` lines

Reply:
@david-oc-miller, I will remove these commits from this pull request